src/main/java/com/cinema/service/impl/OrderServiceImpl.java

```diff
@@ -30,7 +30,7 @@ public Order completeOrder(ShoppingCart shoppingCart) {
         order.setOrderDate(LocalDateTime.now());
         orderDao.add(order);
         shoppingCart.setTickets(new ArrayList<>());
```

Comment:
do we need this?

Reply:
yes, without it calling .clear() deletes all tickets from orders